test/parallel/test-http2-respond-file-fd-range.js

```diff
@@ -3,19 +3,19 @@
 // Tests the ability to minimally request a byte range with respondWithFD
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: would you mind moving this after the `common.hasCrypto` check?

Reply:
moved!